apps/server/package.json

```diff
@@ -48,6 +48,7 @@
     "axios": "^1.6.8",
     "buffer-browserify": "^0.2.5",
     "cookie-parser": "^1.4.7",
+    "cors": "^2.8.5",
```

Comment:
Adding cors allowed me to use the swagger preview in VSCode to hit our api through the openapi spec which I thought could be helpful for people who don't have postman setup.